<?xml version="1.0"?>
<launch>

	<group ns = "box_entry_gate_detection">
		<param name="execute_default" type="bool" value="false"/>
		<param name="pub_plane_cloud" type="bool" value="false"/>

		<param name="sub_point_topic_name" type="str" value="/hsrb/head_rgbd_sensor/depth/points"/>

		<param name="base_frame_name" type="str" value="/base_footprint"/>
		<param name="map_frame_name" type="str" value="/map"/>

		<param name="depth_range_min_x" type="double" value="0.1"/>
		<param name="depth_range_max_x" type="double" value="1.2"/>
		<param name="depth_range_min_z" type="double" value="0.1"/>
		<param name="depth_range_max_z" type="double" value="1.2"/><!--1.6-->
		<param name="cluster_ss" type="double" value="0.05"/>

		<node name="box_detection_node" pkg="box_entry_gate_detection" type="box_detect"  output="screen"/>

	</group>

</launch>
